lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -15,6 +15,16 @@
  * @access private
  */
 class WP_Theme_JSON_6_1 extends WP_Theme_JSON_6_0 {
+
+	/**
+	 * Whitelist which defines which pseudo selectors are enabled for
+	 * which elements.
+	 * Note: this will effect both top level and block level elements.
+	 */
+	const VALID_ELEMENT_PSEUDO_SELECTORS = array(
+		'link' => array( ':hover', ':focus' ),
```

Comment:
in addition to these, most themes style `:active` state too. It would be great to add it to allow list.

Reply:
Yeh I guess that wouldn't be too much of an issue.